reference/OpenAPI.json

```diff
@@ -24245,93 +24245,247 @@
+                "expires_at"
+              ]
+            }
+          },
+          "filters": {
+            "$ref": "#/components/schemas/LoyaltiesPointsExpirationExportCreateResponseBodyParametersFilters"
           }
         }
       },
-      "8_req_export_points_expirations": {
-        "title": "Points Expirations Export Request Body",
+      "LoyaltiesPointsExpirationExportCreateRequestBodyParametersFilters": {
         "type": "object",
-        "description": "Request body schema for creating points expirations. ",
+        "description": "Data filters used to narrow the data records to be returned in the export.",
         "properties": {
-          "parameters": {
+          "junction": {
+            "$ref": "#/components/schemas/Junction"
+          },
+          "voucher_id": {
             "type": "object",
-            "description": "List of fields and filters that were passed in the request body to create the export.",
+            "description": "Data filters used to narrow the data records to be returned in the result.",
             "properties": {
-              "order": {
-                "type": "string",
-                "description": "How the export is filtered, where the dash `-` preceding a sorting option means sorting in a descending order.",
-                "enum": [
-                  "expires_at",
-                  "-expires_at"
-                ]
-              },
-              "fields": {
-                "type": "array",
-                "description": "Array of strings containing the data that was exported. These fields define the headers in the CSV file. \n\n The array can be a combination of any of the following available fields:  \n\n| **Field** | **Definition** | **Example Export** |\n|:---|:---|:---|\n| id | Loyalty points bucket ID. | lopb_Wl1o3EjJIHSNjvO5BDLy4z1n |\n| campaign_id | Campaign ID of the parent loyalty campaign. | camp_7s3uXI44aKfIk5IhmeOPr6ic |\n| voucher_id | Voucher ID of the parent loyalty card. | v_YLn0WVWXSXbUfDvxgrgUbtfJ3SQIY655 |\n| status | Status of the loyalty points bucket. | `ACTIVE` or `INACTIVE` |\n| expires_at | Timestamp in ISO 8601 format representing the date when the points expire. | 2022-06-30 |\n| points | Number of points. | 1000 |",
-                "items": {
-                  "type": "string"
+              "conditions": {
+                "type": "object",
+                "description": "Data filters used to narrow the data records to be returned in the result.",
+                "properties": {
+                  "$in": {
+                    "type": "array",
+                    "description": "Array of resource values that should be included in the results.",
+                    "items": {
+                      "type": "string"
+                    }
+                  },
+                  "$not_in": {
+                    "type": "array",
+                    "description": "Array of resource values that should be included in the results.",
```

Comment:
`"description": "Array of the resource values that should be excluded from the results.",`